### DISCUSSION TITLE
New scene not indenting in project view

### DISCUSSION THREAD

**Discussion by mogplus88** [vkbo/novelWriter#1530] [Help]
From a few days ago NW is not inserting new scenes in the project view correctly. They still look right in the outline view but not in the project view. Also the heading at the top of the page says "Novel > New Scene", not "Novel > chapter name > scene name" as it did before, and still does for chapters I have already written. It also works correctly if I add a new scene to an old chapter that already has some scenes in it correctly indented. It's only new chapters that have this problem. Build still work correctly.
I tried downloading the .deb file again and reinstalling but the problem remains. I also tried the appimage but that did the same thing too.
Am I doing something wrong?
I'm using Linux Mint, latest version.
Thanks.

![Screenshot from 2023-10-04 11-42-15](https://github.com/vkbo/novelWriter/assets/15082808/76c18089-3756-4de1-854f-63d5d4be711b)

**Comment by vkbo** [comment]
Just drag the scene and drop it on the chapter to put it under the chapter.

**Reply by vkbo** [reply]
Just to clarify, the logic is so that if you select a chapter and add a new scene, the scene will only be added as a child of the chapter if the chapter already has child elements. Otherwise it will be added after the chapter.  So the first time you need to add the scene as a child of the chapter manually. After that, it changes behaviour.

It was designed that way because the app doesn't actually know which option you want the first time, so it falls back to the pre version 2.0 behaviour of adding it after.
